tests/integration/targets/dcnm_vrf/tests/dcnm/merged.yaml

```diff
@@ -204,7 +204,14 @@
     fabric: "{{ test_fabric }}"
     state: deleted
 
-- name: MERGED - Create, Attach and Deploy new VRF - VRF/VRF LITE EXTENSION Provided by the DCNM - Only Mandatory option - Rest are populated from DCNM
+- name: QUERY - sleep for 40 seconds for DCNM to completely remove lite profile
```

Comment:
Can we update the tests to use `retry` instead similar to https://github.com/CiscoDevNet/ansible-dcnm/blob/develop/tests/integration/targets/dcnm_network/tests/dcnm/self-contained-tests/merged_net_all.yaml#L36

Let's try to remove arbitrary sleep whenever possible

Reply:
This can be removed after check_deploy support, which is planned for a later release.